src/utils/viewer/viewer.cpp

```diff
@@ -180,9 +176,13 @@ Viewer::Viewer(const Arguments& arguments)
   rgbSensorNode_->translate({0.0f, rgbSensorHeight, 0.0f});
   agentBodyNode_->translate({0.0f, 0.0f, 5.0f});
 
-  int width = viewportSize[0];
-  int height = viewportSize[1];
-  renderCamera_->setProjectionMatrix(width, height, znear_, zfar_, hfov_);
+  renderCamera_->setProjectionMatrix(viewportSize.x(),  // width
+                                     viewportSize.y(),  // height
```

Comment:
Can I keep the comment here so that we understand the order of input parameters in the function? That means we know here is w x h, not h x w.

Reply:
Sure, it was just an idea how to make it "self-documenting".